src/app/_components/team/joinTeam.tsx

```diff
@@ -0,0 +1,17 @@
+'use client';
+
+import {joinTeamAction} from "~/app/teams/join/[TEAM_ID]/actions";
+
+export default function JoinTeam({
```

Comment:
Also, I'd rather not make this a separate component for now. Maybe just use the joinTeamAction callback in the component that uses this?

Reply:
Event handlers cannot be passed to client component props, Next docs recommend converting part of this (in this case the button) to a client component